app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(
+        elves: list[Elf]
```

Comment:
There is only one parameter, so no need to place it on the new line. Please correct